src/phoenix/db/models.py

```diff
@@ -490,3 +490,58 @@ class DatasetExampleRevision(Base):
             "dataset_version_id",
         ),
     )
+
+
+class Experiment(Base):
+    __tablename__ = "experiments"
+    id: Mapped[int] = mapped_column(primary_key=True)
+    dataset_id: Mapped[int] = mapped_column(
+        ForeignKey("datasets.id", ondelete="CASCADE"),
+        index=True,
+    )
+    dataset_version_id: Mapped[int] = mapped_column(
+        ForeignKey("dataset_versions.id", ondelete="CASCADE"),
+        index=True,
+    )
+    metadata_: Mapped[Dict[str, Any]] = mapped_column("metadata")
+    created_at: Mapped[datetime] = mapped_column(UtcTimeStamp, server_default=func.now())
```

Comment:
I think we might want updated_at here too, since we'll patch to complete the experiment, I imagine we might want an additional column to indicate its status too?

Reply:
addressed in af62e03d3193920e134359fb27d57cfc91b9b003